packages/@coorpacademy-components/src/molecule/video-player/jwplayer.js

```diff
@@ -36,7 +36,7 @@ class JWPlayer extends React.Component {
     const changes = keys(this.props).filter(_name => this.props[_name] !== prevProps[_name]);
     const shouldStart = includes('autoplay', changes);
     if (shouldStart) {
-      if (window.jwplayer) {
+      if (window.jwplayer && typeof window.jwplayer === 'function') {
```

Comment:
vu que d'après la ligne 37 tu as lodash, autant faire un simple:
```suggestion
      if (isFunction(window.jwplayer) {
```
(en récupérant bien sur la fonction dans les imports)